localisation/00_zuwc_united_localisation.csv

```diff
@@ -2849,7 +2845,7 @@ GetMonstrousUndeadNameDesc_WickerGolem;a hulking creation that is crafted from m
+EVTDESC_WCCRS_110;Recently, the illness, that has befallen [target_sick_spouse.GetFirstName] has been growing worse with each passing day. Knowing well that [target_sick_spouse.GetSheHe] might not last another week, you have sought many solutions, be it magical or alchemical, but nothing seems to help alleviate [target_sick_spouse.GetHerHis] condition. The grief over your inevitable loss overcomes you, and in desperation you cry out in hopes that something or someone will hear your pleas. To your surprise, a shadowy figure emerges and makes you a promise that death will never part you and your spouse... for a price. It offers you unimaginable power and eternal life in exchange for loyalty and you realise this might be the only way to keep your beloved at your side. Conflicting thoughts cross your mind, but you ultimately realise that if death can part you both...;;In letzter Zeit hat sich die Krankheit, die [target_sick_spouse.GetFirstName] befallen hat, mit jedem Tag verschlimmert. Wohl wissend, dass [target_sick_spouse.GetSheHe] vielleicht keine weitere Woche �berdauern wird, habt Ihr viele L�sungen gesucht, seien sie magischer oder alchemistischer Natur, aber nichts scheint [target_sick_spouse.GetHerHis]en Zustand zu lindern. Die Trauer �ber den unvermeidlichen Verlust �berkommt Euch, und in Eurer Verzweiflung schreit Ihr in der Hoffnung, dass etwas oder jemand Eure Bitten erh�rt. Zu Eurer �berraschung taucht eine schattenhafte Gestalt auf und verspricht Euch, dass der Tod Euch und Eure[target_sick_spouse.GetnMale] Gemahl[target_sick_spouse.GetinFemaleOpp] niemals trennen wird... f�r einen Preis. Sie bietet Euch unvorstellbare Macht und ewiges Leben im Tausch gegen Loyalit�t an und Ihr erkennt, dass dies der einzige Weg sein k�nnte, Eure[target_sick_spouse.GetnMale] Geliebte[target_sick_spouse.GetnMale] an Eurer Seite zu halten. Widerstreitende Gedanken gehen Euch durch den Kopf, aber letztlich erkennt Ihr, dass, wenn der Tod Euch beide trennen kann...;;;;;;;;;;;x
```

Comment:
Remove unnecessary comma in the first sentence.
```suggestion
EVTDESC_WCCRS_110;Recently, the illness that has befallen [target_sick_spouse.GetFirstName] has been growing worse with each passing day. Knowing well that [target_sick_spouse.GetSheHe] might not last another week, you have sought many solutions, be it magical or alchemical, but nothing seems to help alleviate [target_sick_spouse.GetHerHis] condition. The grief over your inevitable loss overcomes you, and in desperation you cry out in hopes that something or someone will hear your pleas. To your surprise, a shadowy figure emerges and makes you a promise that death will never part you and your spouse... for a price. It offers you unimaginable power and eternal life in exchange for loyalty and you realise this might be the only way to keep your beloved at your side. Conflicting thoughts cross your mind, but you ultimately realise that if death can part you both...;;In letzter Zeit hat sich die Krankheit, die [target_sick_spouse.GetFirstName] befallen hat, mit jedem Tag verschlimmert. Wohl wissend, dass [target_sick_spouse.GetSheHe] vielleicht keine weitere Woche �berdauern wird, habt Ihr viele L�sungen gesucht, seien sie magischer oder alchemistischer Natur, aber nichts scheint [target_sick_spouse.GetHerHis]en Zustand zu lindern. Die Trauer �ber den unvermeidlichen Verlust �berkommt Euch, und in Eurer Verzweiflung schreit Ihr in der Hoffnung, dass etwas oder jemand Eure Bitten erh�rt. Zu Eurer �berraschung taucht eine schattenhafte Gestalt auf und verspricht Euch, dass der Tod Euch und Eure[target_sick_spouse.GetnMale] Gemahl[target_sick_spouse.GetinFemaleOpp] niemals trennen wird... f�r einen Preis. Sie bietet Euch unvorstellbare Macht und ewiges Leben im Tausch gegen Loyalit�t an und Ihr erkennt, dass dies der einzige Weg sein k�nnte, Eure[target_sick_spouse.GetnMale] Geliebte[target_sick_spouse.GetnMale] an Eurer Seite zu halten. Widerstreitende Gedanken gehen Euch durch den Kopf, aber letztlich erkennt Ihr, dass, wenn der Tod Euch beide trennen kann...;;;;;;;;;;;x
```

Reply:
changed